manifests/config.pp

```diff
@@ -10,13 +10,87 @@
     system => true,
   }
 
+  case $::osfamily {
+    'RedHat':  {
+      $nologin_shell = '/sbin/nologin'
+      case $::operatingsystemmajrelease {
```

Comment:
facts['os']['release']['major']

Reply:
Fixed in 2e3af3a4f789bd6d8813dea0fa43de6c2cc243ca